apis/v1alpha1/generator.yaml

```diff
@@ -346,3 +346,12 @@ resources:
     fields:
       Name:
         is_primary_key: true
+  DBProxy:
+    exceptions:
+      terminal_codes:
+        - DBProxyAlreadyExistsFault
+        - DBProxyQuotaExceededFault
+        - InvalidSubnet
+    fields:
+      DBProxyName:
```

Comment:
Did you want to rename this to just `Name` to match the other RDS resources?

Reply:
I think it's ok for now. previously we do the rename to avoid duplicate between spec and status, which is not an issue for DBProxy CRD here